src/ripple/rpc/impl/LookupLedger.cpp

```diff
@@ -25,6 +25,23 @@ namespace RPC {
 
 namespace {
 
+/*  Returns true if the server is stand alone or we think the ledger is
+    historical or we have a recent validated ledger and the ledger's ID is at
+    least 10 larger than the validated. The value 10 is not particularly
+    special here, just some space between the ledgers.
+*/
+bool
+synced (NetworkOPs& netOps, Ledger::pointer ledger)
+{
+    return getConfig ().RUN_STANDALONE ||
```

Comment:
When I originally looked at this function, I was able to interpret using the formatting and grouping. That said, I agree that this is a bit clearer, and may help us in the long run if the logic every has to change.